workers/gossip-server/src/peer_discovery/peers.rs

```diff
@@ -15,6 +15,13 @@ use crate::{
     server::GossipProtocolExecutor,
 };
 
+/// Metric describing the number of local peers the relayer
+/// is connected to
+const NUM_LOCAL_PEERS_METRIC: &str = "num_local_peers";
+/// Metric describing the number of remote peers the relayer
+/// is connected to
+const NUM_REMOTE_PEERS_METRIC: &str = "num_remote_peers";
```

Comment:
why do we centralize metrics names for other metrics, but define them locally in this case? I feel we should do one or the other. And if we make `renegade-metrics` its own crate, it'd probably make sense to centralize them there